### ISSUE TITLE
latest version(2.1.6) still need rosetta

### ISSUE THREAD

**Issue by fyeeme** [oracle/visualvm#497]
**Describe the bug**
when install the latest version 2.1.6, Double-clicking the icon reminded me that I still need to install Rosetta.

**To Reproduce**
1. Go to 'application'
2. Click on 'VisualVM.app'
3. See error

**Expected behavior**
VisualVm should works normal.

**VisualVM log**
no log

**Screenshots**
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/6078758/235893208-089591ac-5f28-4c37-b532-d4b01fb7801d.png">

**Desktop (please complete the following information):**
- Chip: Apple M1 Pro
 - OS: MacOS 13.3.1 (a) (22E772610a)
 - JDK version [17.0.7+8-LTS-224]
 - Version [2.1.6]

**Comment by thurka** [comment]
No, it does not need Rosseta. Rosseta is needed by JDK. You need to run VisualVM on correct JDK - the one build for M1. What is the output of `/usr/libexec/java_home -V` on your machine?

**Comment by fyeeme** [comment]
@thurka 
all of the version on my mac is arm64 version ,and default version is jdk 17
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/6078758/236090367-24658479-2750-4cfa-a0fe-a388cd8c8882.png">

**Comment by thurka** [comment]
This looks OK. Can you please try to run VisualVM from Terminal with the following command and post the output?

```
bash -x /Applications/VisualVM.app/Contents/MacOS/visualvm --jdkhome /Library/Java/JavaVirtualMachines/jdk-17.0.7.jdk/Contents/Home
```

This way we could be able to see, which executable needs Rosetta.

**Comment by fyeeme** [comment]
@thurka  ths for your reply. this way works and the visualvm started successfully.
here is the log , and i still get no using msg why  with --jdkhome params , the visualvm can start on my mac.

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/6078758/236417362-3858a166-d9f7-4598-8778-bd2fb1991b70.png">
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/6078758/236417433-b64feb04-fa62-403e-b14d-4604065d64d0.png">
<img width="1505" alt="image" src="https://user-images.githubusercontent.com/6078758/236417480-6761845e-c8cb-421e-ad87-9ae6cb037a6c.png">

**Comment by thurka** [comment]
> this way works and the visualvm started successfully.

Hmm, too bad that this way you cannot reproduce it. The log does not provide any insigth, what is goign on.

**Comment by thurka** [comment]
Can you see if removing `--jdkhome` option makes any difference? 
 ```
bash -x /Applications/VisualVM.app/Contents/MacOS/visualvm
```

**Comment by thurka** [comment]
Thanks for testing. I am not sure what is going on. Unfortunately currently I cannot test it on M1 without Rosetta 2. Since Rosetta 2 cannot be removed/disable easily. I will keep you posted.

**Comment by thurka** [comment]
I can confirm the issue. Finder seems to be somehow confused and thinks that VisualVM is Intel only application.

**Comment by fyeeme** [comment]
> I can confirm the issue. Finder seems to be somehow confused and thinks that VisualVM is Intel only application.

Thank you for your continued attention to this issue. I have tested it on macOS 14.0, and the problem still persists.
<img width="548" alt="image" src="https://github.com/oracle/visualvm/assets/6078758/eaedaf46-963f-4675-89e6-3e854929f41f">

**Comment by cfecherolle** [comment]
I have the same issue on a Macbook Pro M2. Is there a workaround to start VisualVM without enabling Rosetta?

**Comment by thurka** [comment]
> I have the same issue on a Macbook Pro M2. Is there a workaround to start VisualVM without enabling Rosetta?

Yes, there is workaround mentioned above - just run VisualVM from Terminal using:
```
/Applications/VisualVM.app/Contents/MacOS/visualvm
```

**Comment by cfecherolle** [comment]
It worked, thank you!

**Comment by Skyxim** [comment]
I also encountered the same problem in 2.1.7. Is there a definite fix time?

**Comment by thurka** [comment]
It should be fixed in VisualVM 2.1.8.

**Comment by cfecherolle** [comment]
I can confirm it now works, thank you 👍
